code/datums/components/storage/concrete/_concrete.dm

```diff
@@ -187,6 +187,7 @@
 			parent.add_fingerprint(M)
 			if(!prevent_warning)
 				mob_item_insertion_feedback(usr, M, I)
+	I.item_flags |= IN_INVENTORY

```

Comment:
This completely changes flag meaning and breaks it tracking.

Reply:
Oh damn, you are right. I created IN_STORAGE define instead so it has proper flag meaning and does not break IN_INVENTORY.